tests/messaging.test.js

```diff
@@ -0,0 +1,45 @@
+import { Client, ApiController, FileWrapperi, MessagingExceptionError, MessageRequest } from '../src';
+
+let controller;
+
+beforeEach(() => {
+    const client = new Client({
+        basicAuthUserName: process.env.BANDWIDTH_USERNAME,
+        basicAuthPassword: process.env.BANDWIDTH_PASSWORD
+    });
+
+    controller = new ApiController(client);
+});
+
+describe('messaging', () => {
+    
+    it('should create message with proper values', async () => {
+        const accountId = process.env.BANDWIDTH_ACCOUNT_ID;
+        const body = {
+            applicationId: process.env.MESSAGING_APPLICATION_ID,
+            to: [process.env.PHONE_NUMBER_INBOUND],
+            from: process.env.PHONE_NUMBER_OUTBOUND,
+            text: 'TS messaging test'
+        };
+
+        const createMessageResponse = await controller.createMessage(accountId, body);
+        expect(createMessageResponse.result.applicationId).toBeDefined();
```

Comment:
I'd do a comparison to validate that the returned application id is the expected one and toss a few more expectations in there for the rest of the response values.

Reply:
Oh you're right, we can check the application ID since that's something that will be consistent. Updated

Reply:
I'd add the rest of the expected response fields in there as well. Might as well check everything in one swoop!

Reply:
Good callout, done 🎉